how-to/integrate-with-sso/integrate-with-okta/public/widgetLogin.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no" />
+		<link
+			rel="stylesheet"
+			href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+			integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk"
+			crossorigin="anonymous"
+		/>
+		<title>Simple Web Page</title>
+		<!-- Latest CDN production JavaScript and CSS -->
+		<script
+			src="https://global.oktacdn.com/okta-signin-widget/7.2.2/js/okta-sign-in.min.js"
+			type="text/javascript"
+		></script>
+		<link
+			href="https://global.oktacdn.com/okta-signin-widget/7.2.2/css/okta-sign-in.min.css"
+			type="text/css"
+			rel="stylesheet"
+		/>
+		<style>
+			h1 {
+				margin: 2em 0;
+			}
+		</style>
+		<!-- widget stuff here -->
+		<script
+			src="https://global.oktacdn.com/okta-signin-widget/${widgetVersion}/js/okta-sign-in.min.js"
+			type="text/javascript"
+		></script>
+		<link
+			href="https://global.oktacdn.com/okta-signin-widget/${widgetVersion}/css/okta-sign-in.min.css"
+			type="text/css"
+			rel="stylesheet"
+		/>
+	</head>
+	<body>
+		<div class="container">
+			<h1 class="text-center">Simple Web Page</h1>
+			<div class="text-center" style="color: red" id="messageBox">
+				<h2>You are not logged in.</h2>
+			</div>
+			<!-- where the sign-in form appears -->
+			<div id="okta-login-container"></div>
+			<button id="logout" class="button" onclick="logout()" style="display: none">Logout</button>
+		</div>
+		<script type="text/javascript">
+			var oktaConfig = {
+				features: {
+					registration: true, // Enable self-service registration flow
+					rememberMe: false, // Setting to false will remove the checkbox to save username
+					//multiOptionalFactorEnroll: true,            // Allow users to enroll in multiple optional factors before finishing the authentication flow.
+					//selfServiceUnlock: true,                    // Will enable unlock in addition to forgotten password
+					//smsRecovery: true,                          // Enable SMS-based account recovery
+					//callRecovery: true,                         // Enable voice call-based account recovery
+					router: true // Leave this set to true for this demo
+				},
+				logo: 'http://localhost:8080/favicon.ico',
+				issuer: '{YOUR_OKTA_DOMAIN}/oauth2/default',
+				redirectUri: 'http://localhost:8080/platform/app.html',
```

Comment:
The flow here it not quite correct, it should be:
* App Shows login dialog
* On login it redirects to login-complete.html with the token in the query params
* The platform polls the login window to see when it has redirected to login-complete, and extracts the token
* The platform then launches the app window